src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -94,226 +112,67 @@ private async Task WriteSolutionFiles()
-
-## Next steps
-
-* Provide a link to additional code examples, ideally to those sitting alongside the README in the package's `/samples` directory.
-* If appropriate, point users to other packages that might be useful.
-* If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
-
-## Contributing
-
-This is a template, but your SDK readme should include details on how to contribute code to the repo/package.
-
-<!-- LINKS -->
-[style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
-[style-guide-cloud]: https://aka.ms/azsdk/cloud-style-guide
-
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net/sdk/{1}/{0}/README.png)
-";
+            string multipleApiVersionContent = GetAssetContent("Readme_multipleApiVersionContent_DataPlane.md");
+            string readmeContent = GetAssetContent("Readme_readmeContent_DataPlane.md");
             return string.Format(readmeContent, Configuration.Namespace, _serviceDirectoryName, (Configuration.AzureArm || Configuration.Generation1ConvenienceClient) ? "" : multipleApiVersionContent);
         }
 
         private string GetCiYml()
         {
             string safeName = Configuration.Namespace.Replace(".", "");
-            const string ciYmlContent = @"# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
-
-trigger:
-  branches:
-    include:
-    - main
-    - hotfix/*
-    - release/*
-  paths:
-    include:
-    - sdk/{0}
-    - sdk/{0}/ci.yml
-    - sdk/{0}/{1}
-
-pr:
-  branches:
-    include:
-    - main
-    - feature/*
-    - hotfix/*
-    - release/*
-  paths:
-    include:
-    - sdk/{0}
-    - sdk/{0}/ci.yml
-    - sdk/{0}/{1}
-
-extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
-  parameters:
-    ServiceDirectory: {0}
-    ArtifactName: packages
-    Artifacts:
-    - name: {1}
-      safeName: {2}
-";
+            string ciYmlContent = GetAssetContent("ci.yml");
             return string.Format(ciYmlContent, _serviceDirectoryName, Configuration.Namespace, safeName);
         }
 
         private string GetDirectoryBuildProps()
         {
-            const string directoryBuildPropsContent = @"<Project ToolsVersion=""15.0"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
-  <!--
-    Add any shared properties you want for the projects under this package directory that need to be set before the auto imported Directory.Build.props
-  -->
-  <Import Project=""$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props"" />
-</Project>
-";
+            string directoryBuildPropsContent = GetAssetContent("Directory.Build.props");
```

Comment:
can we just call `GetAssetContent("Directory.Build.props")` instead of a new method?